site/components/Transactions.js

```diff
@@ -0,0 +1,29 @@
+import { useContext } from 'react'
+
+import TransactionsContext from './context/Transactions'
+import TransactionsModal from './TransactionsModal'
+
+function Transactions() {
+  const { currentTransactions, modalIsOpen, closeModal, openedOpId } =
+    useContext(TransactionsContext)
+
+  if (!currentTransactions || !currentTransactions.length) {
+    return null
+  }
+
+  const transaction = currentTransactions.find((t) => t.opId === openedOpId)
+  if (!transaction) {
+    return null
+  }
+
+  return (
+    <TransactionsModal
+      closeModal={closeModal}
+      key={openedOpId}
```

Comment:
`key` is not required here, as this is not being rendered in an array


```suggestion
```

Reply:
Fixed in b73e76b.